java/test/src/main/java/test/Ice/idleTimeout/Test.ice

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) ZeroC, Inc.
+
+#pragma once
+
+[["java:package:test.Ice.idleTimeout"]]
+
+module Test
+{
+    sequence<byte> ByteSeq;
```

Comment:
Why do we have this `ByteSeq` here? it seems we do the same in C#.

Reply:
Removed in Java and C#.